include/openssl/core_dispatch.h

```diff
@@ -412,6 +415,16 @@ OSSL_CORE_MAKE_FUNC(const OSSL_PARAM *, cipher_settable_ctx_params,
                     (void *cctx, void *provctx))
 OSSL_CORE_MAKE_FUNC(const OSSL_PARAM *, cipher_gettable_ctx_params,
                     (void *cctx, void *provctx))
+OSSL_CORE_MAKE_FUNC(int, cipher_encrypt_skey_init, (void *cctx,
+                                                    void *skeydata,
+                                                    const unsigned char *iv,
+                                                    size_t ivlen,
+                                                    const OSSL_PARAM params[]))
+OSSL_CORE_MAKE_FUNC(int, cipher_decrypt_skey_init, (void *cctx,
+                                                    void *skeydata,
```

Comment:
Should the SKEY pointers be `const void *skey`?

Reply:
I can imagine the PKCS#11 based scenario when we have to revive the handle

Reply:
@t8m we may delay actual loading of internal data of a key in some situation, or as @beldmit pints out need to perform a refresh because a fork (or something else) happened in between calls ... In general it is under the provider control what are the contents of skeydata and the calling application should not make assumptions.

Reply:
We only guarantee that the pointer will remain valid until skeymgmt->free() is called on it.